src/libraries/System.Private.CoreLib/src/System/Collections/Hashtable.cs

```diff
@@ -798,7 +798,7 @@ protected virtual bool KeyEquals(object? item, object key)
 
             if (_keycomparer != null)
                 return _keycomparer.Equals(item, key);
-            return item == null ? false : item.Equals(key);
+            return item != null && item.Equals(key);
```

Comment:
These could be `item is not null &&` or possibly `item?.Equals(key)`, yes?

Reply:
> These could be `item is not null &&`

I chose to match the existing code style in the file.

> or possibly `item?.Equals(key)`, yes?

`item?.Equals(key) == true` seemed less a little less clear.